src/main/play-doc/operation/Install.md

```diff
@@ -1235,7 +1245,7 @@ frontend conductr-haproxy-test
   mode http
   monitor-uri /test
 
-{{#eachAcls bundles defaultHttpPort=9000}}
+{{#eachAcls bundles defaultHttpPort=80}}
```

Comment:
Good question. I moved to a range mapping, but didn't want to tie up so
much using 9000-9999, so I did 9099-9999.

On Wed, Aug 24, 2016, 17:38 Felix Satyaputra <notifications@github.com>
wrote:

> In src/main/play-doc/operation/Install.md
> <https://github.com/typesafehub/conductr-doc/pull/255#discussion_r76164417>
> :
>
> > @@ -1235,7 +1245,7 @@ frontend conductr-haproxy-test
> >    mode http
> >    monitor-uri /test
> >
> > -{{#eachAcls bundles defaultHttpPort=9000}}
> > +{{#eachAcls bundles defaultHttpPort=80}}
>
> I think our default template exposes HTTP based service via the proxy on
> port 9000. Why do we change it to port 80?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/typesafehub/conductr-doc/pull/255/files/f123f9fe47e4e16ba4df1cd587fb2e9fcc4b8270#r76164417>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwCo54bq0q1mRG8bzH_LEB9VXFbIjMeks5qjOQVgaJpZM4JslBg>
> .
>